types/react-bootstrap-typeahead/index.d.ts

```diff
@@ -424,7 +424,7 @@ export const Loader: React.FunctionComponent<LoaderProps>;
 --------------------------------------------------------------------------- */
 export interface AutosizeInputProps extends Pick<React.InputHTMLAttributes<HTMLInputElement>, "className" | "style"> {
     inputClassName?: string | undefined;
-    inputRef?: React.LegacyRef<HTMLInputElement> | undefined;
```

Comment:
This was likely a bug. If you'd pass it as a string, the owner of that string ref would not see it in `this.refs`. `AutoSizeInput` would see it in `this.refs` though if it passes `inputRef` into a `ref` prop and if `AutoSizeInput` is a class component.